Makefile

```diff
@@ -31,7 +31,27 @@ ifeq ($(GOOS),windows)
 	IS_EXE := .exe
 endif
 MINISHIFT_BINARY ?= $(GOPATH)/bin/minishift$(IS_EXE)
-GODOG_OPTS ?= ""
+ifeq ($(filter-out integration,$(MAKECMDGOALS)),all)
```

Comment:
Filter out the goal ```integration``` and compare with the arguments 
(make integration all,all) -> (all,all)